posthog/models/activity_logging/activity_log.py

```diff
@@ -104,25 +105,32 @@ def log_activity(
     activity: str,
     detail: Detail,
 ) -> None:
-    if activity == "updated" and (detail.changes is None or len(detail.changes) == 0):
-        logger.warn(
-            "ignore_update_activity_no_changes",
-            team_id=team_id,
+    try:
+        if activity == "updated" and (detail.changes is None or len(detail.changes) == 0):
+            logger.warn(
+                "ignore_update_activity_no_changes",
+                team_id=team_id,
+                organization_id=organization_id,
+                user_id=user.id,
+                scope=scope,
+            )
+            return
+
+        ActivityLog.objects.create(
             organization_id=organization_id,
-            user_id=user.id,
+            team_id=team_id,
+            user=user,
+            item_id=str(item_id),
             scope=scope,
+            activity=activity,
+            detail=detail,
         )
-        return
-
-    ActivityLog.objects.create(
-        organization_id=organization_id,
-        team_id=team_id,
-        user=user,
-        item_id=str(item_id),
-        scope=scope,
-        activity=activity,
-        detail=detail,
-    )
+    except Exception as e:
+        with push_scope() as sentry_scope:
+            sentry_scope.set_tag("team-id", team_id)
+            sentry_scope.set_tag("scope", scope)
+            sentry_scope.set_tag("activity", activity)
+            capture_exception(e)
```

Comment:
Does this also log to the console? I know there was a comment re. sentry, but would be good to ensure it also logs

Reply:
I shouldn't have to look in sentry to understand what is going on ideally, the logs should give a reasonable picture on it's own. I'm not, for instance going to have sentry locally, and I'm not going to be able to easily correlate with sentry the surrounding context.

Reply:
It does both now :)

Reply:
Arguably the human shouldn't have to care about this... It looks like sentry can be set to capture any logged error https://docs.sentry.io/platforms/python/guides/logging/ but I'm not biting that off here :)

Reply:
Ahhhhh

Reply:
It did feel like a lot of instrumentation for one method call :)